docs/consume-packages/consuming-packages-authenticated-feeds.md

```diff
@@ -3,98 +3,102 @@ title: Consuming packages from authenticated feeds
 description: Consuming packages from authenticated feeds in all NuGet client scenarios
 author: nkolev92
 ms.author: nikolev
-ms.date: 02/28/2020
+ms.date: 12/22/2023
 ms.topic: conceptual
 ---
 
 # Consuming packages from authenticated feeds
 
-In addition to the nuget.org [public feed](https://api.nuget.org/v3/index.json), NuGet clients have the ability to interact with file feeds and private http feeds.
+Many NuGet operations, such as restore and install, require communication with one or more package sources, which [can be configured in *nuget.config* files](../reference/nuget-config-file.md#packagesources).
+For HTTP feeds, NuGet will with make an unauthenticated request, and if the server responds with an HTTP 401 response, NuGet will search for credentials in the following order:
 
+1. [An environment variable `NuGetPackageSourceCredentials_{name}`](#credentials-in-environment-variables).
+1. [Add credentials in a *nuget.config* file](#credentials-in-nugetconfig-files).
+1. [Use a NuGet credential provider, if your package source provides one](#credential-providers).
 
-To authenticate with private http feeds, the 2 approaches are:
+We recommend using a credential provider when possible.

```

Comment:
I feel like this should have a larger emphasis.

Reply:
> [!NOTE]
> We recommend using a credential provider when possible.